package.json

```diff
@@ -3,9 +3,10 @@
     "@angular/common": "2.0.0-rc.3",
     "@angular/compiler": "2.0.0-rc.3",
     "@angular/core": "2.0.0-rc.3",
+    "@angular/http": "2.0.0-rc.3",
     "@angular/platform-browser": "2.0.0-rc.3",
     "@angular/platform-browser-dynamic": "2.0.0-rc.3",
-    "@angular/http": "2.0.0-rc.3",
+    "angular2-jwt": "^0.1.16",
```

Comment:
New dependencies so they will need to be installed after this is merged: `npm i` should do it.